src/Component.elm

```diff
@@ -1,9 +1,32 @@
-module Component exposing (materialIcon)
+module Component exposing (empty, materialIcon, progressBar)
 
 import Html as H
 import Html.Attributes as HA
+import RemoteData
+
+
+empty : H.Html msg
+empty =
+    H.div [] []
 
 
 materialIcon : String -> H.Html msg
 materialIcon icon =
     H.i [ HA.class "material-icons" ] [ H.text icon ]
+
+
+progressBar : RemoteData.RemoteData e a -> H.Html msg
+progressBar remoteData =
+    case remoteData of
+        RemoteData.Loading ->
```

Comment:
Http library also contains [progress API](http://package.elm-lang.org/packages/elm-lang/http/1.0.0/Http-Progress). You might be interested to play with it :)

Reply:
This function was meant to be like really small attempt to write a function that accepts `RemoteData` but cares not about `e` and `a` :D. Since it's working, I think it's valuable experience. 

Though I will check `progressApi` ;-)